cypress/objects/timeLogging.js

```diff
@@ -0,0 +1,19 @@
+class TimeLogging {   //klasiu pav is didziosios
```

Comment:
Oh also one more thing: it's great to keep comments, we all do, but when working on actual projects in the future try not to commit such comments that you make for yourself, rather write stuff like this down somewhere else or remove it when you're committing. But it's alright here in our lectures :)